src/serlo-editor/plugins/fill-in-the-blanks-exercise/index.tsx

```diff
@@ -12,6 +12,10 @@ import { EditorPluginType } from '@/serlo-editor/types/editor-plugin-type'
 
 export type FillInTheBlanksMode = 'typing' | 'drag-and-drop'
 
+export type BlankId = string
+
+export type DraggableId = string
+
 export const fillInTheBlanksExercise: EditorPlugin<FillInTheBlanksExerciseState> =
```

Comment:
```suggestion
export const ExerciseWithBlanks: EditorPlugin<FillInTheBlanksExerciseState> =
```

For me `FillInTheBlanks` already includes the mode `typing` -> What is a good name for `fill-in-the-blanks` and `drag-the-words`? `ExerciseWithBlanks` or `BlankExercise`?

Reply:
Hm, `ExerciseWithBlanks` and `BlankExercise` could also for example be an image where you have blanks. 
I changed it to `textWithBlanksExercise` for now but only for the database entries. Component renaming can be done later.

Reply:
Database entries will show `type: blanksExercise` now. Component renaming still to do, but this does not block the first release.